stacker/tests/hooks/test_lambda.py

```diff
@@ -98,6 +98,40 @@ def test_bucket_custom(self):
         self.assert_s3_bucket('custom')
 
     @mock_s3
+    def test_prefix_missing(self):
```

Comment:
I think the two tests here actually both test if a prefix is provided - this test should probably be done with no prefix, right?